tests/test_ext_graphviz.py

```diff
@@ -26,7 +26,7 @@ def decorator(app, *args, **kwargs):
                 dot = subprocess.Popen([graphviz_dot, '-V'],
                                        stdout=subprocess.PIPE,
                                        stderr=subprocess.PIPE)  # show version
-                dot.wait()
```

Comment:
It suppresses `ResourceWarning`:
```
/home/travis/build/sphinx-doc/sphinx/tests/util.py:271: ResourceWarning: unclosed file <_io.BufferedReader name=10>
  func(app, status, warning, *args2, **kwargs2)
```
https://travis-ci.org/sphinx-doc/sphinx/jobs/184006704#L1305

It says the pipes for `Popen()` are not closed on `Popen.wait()`. So I use `Popen.communicate()` to close them politely.